src/components/ProjectionError.js

```diff
@@ -4,32 +4,22 @@ import { useTranslation } from 'react-i18next';
 import { Message } from 'semantic-ui-react';
 import styles from './projectionError.module.css';
 
-const requiredFieldsForProjection = [
```

Comment:
Why did you remove the required field validation?

Reply:
Because last time it was discussed and Uli suggested so in the ticket.